net/https-dns-proxy/files/https-dns-proxy.init

```diff
@@ -110,17 +113,19 @@ service_triggers() {
 
 dnsmasq_add_doh_server() {
 	local cfg="$1" value="$2"
+	uci -q del_list dhcp."$cfg".server="$value"
 	uci -q add_list dhcp."$cfg".server="$value"
 }
 
 dnsmasq_create_server_backup() {
 	local cfg="$1" i
```

Comment:
I think this **i** should go into a separate line so we have **local i**
If we change the code this could be overlook in the future

Reply:
Unless there are other changes required for this PR, how about I make this change locally and it will be a part of the next PR?